resources/documentation/deploy_bluebanquise.rst

```diff
@@ -193,8 +229,14 @@ And execute them, using extra var target to target them:
   ansible-playbook /etc/bluebanquise/login1.yml
   ansible-playbook /etc/bluebanquise/computes.yml --extra-vars "target=c001,c002,c003,c004"
```

Comment:
"target" var must be removed or replaced by -l/--limit option

Reply:
Ok, corrected.